### ISSUE TITLE
Cannot use a component in multiple places

### ISSUE THREAD

**Issue by silentninja** [akxcv/vuera#28]
I tried using the resizable react component in vue. It works great if I use the component only once in a page, if i have more than one instances of the same component in a single page, the component that is created first is being used instead of a new component instance being created.

**Comment by akxcv** [comment]
Hi! Thanks for reporting this issue. Could you please share a code example? It's not clear how to reproduce the issue from your description, I tried rendering multiple React components inside a Vue component, and it worked: https://codesandbox.io/s/qlv7yj5j4j

**Comment by silentninja** [comment]
Multiple component works fine with your example but when using with another library https://github.com/bokuweb/react-rnd The second component is being a duplicate of the first component.

Sorry for sharing a sample code. I  have made changes to the code you put up earlier. 
https://codesandbox.io/s/qlv7yj5j4j 
In the example the messages are reflected properly at first but when you change it in the input box the second component props becomes unresponsive and use the first component's props.

For example if you edit message2 in the input you will notice that the message 2 doesnt get changed, but if you change message 1  input box both message 1 and message 2 will be changed

**Comment by silentninja** [comment]
I think the problem is with the component and not with vuera, thanks a lot for your help.

**Comment by johannes-z** [comment]
@silentninja @akxcv How is this a problem with the component, and how to avoid it?

I have the exact same problem, where I'm using a react component inside a v-for, and each react component has its slot defined, eg.:

```
<react-component v-for="(entry, index) in data" :key="index">
  <div>{{ title }}</div>
</react-component>
```
If entry was `[1, 2, 3]`, this would be the DOM generated:
```
<!-- Before -->
<div>1</div>
<div>2</div>
<div>3</div>

<!-- After -->
<div>1</div>
<div>1</div>
<div>1</div>
```

**Comment by silentninja** [comment]
I tried using a custom react component and it worked fine. The rnd library had the duplication issue and i ended up using a vue drag library
